server/models/userFollower.js

```diff
@@ -0,0 +1,15 @@
+
+
+module.exports = (sequelize, DataTypes) => {
+  const userFollower = sequelize.define('User_follower', {
```

Comment:
`User_Follower` is a `joining` table. It should not be a Model. 

`User` belong to many `User` as `Follower` through the `UserFollower` joining table.